dcrtimed/dcrtimed.go

```diff
@@ -1201,6 +1233,77 @@ func (d *DcrtimeStore) lastAnchorV2(w http.ResponseWriter, r *http.Request) {
 	})
 }
 
+func (d *DcrtimeStore) lastDigestsV2(w http.ResponseWriter, r *http.Request) {
+	defer r.Body.Close()
```

Comment:
This is not a great way to do this. It makes it very easy to forget to close a request body when adding new routes or modifying existing routes, which results in a memory leak.  A better way is to add middleware to the router that closes the request body automatically once the request handler returns.

https://github.com/decred/politeia/blob/master/politeiawww/middleware.go#L19-L25